torch/utils/_sympy/functions.py

```diff
@@ -562,23 +562,37 @@ class MinMaxBase(Expr, LatticeOp):  # type: ignore[misc]
     def __new__(cls, *args, **assumptions):
         from sympy.core.parameters import global_parameters
 
-        evaluate = assumptions.pop("evaluate", global_parameters.evaluate)
-        args = (sympify(arg) for arg in args)
+        import torch._dynamo.config as config
 
-        # first standard filter, for cls.zero and cls.identity
-        # also reshape Max(a, Max(b, c)) to Max(a, b, c)
+        evaluate = assumptions.pop("evaluate", global_parameters.evaluate)
+        args = [sympify(arg) for arg in args]
```

Comment:
became a list instead of generator 🤔

Reply:
was trying to avid typing error but this even did not fix it, but since it did not introduce regression i kept it as is, but i will revert it back . this is not needed.